otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css

```diff
@@ -232,6 +232,11 @@ body {
   font-size: 1.5rem;
   text-align: center;
   flex-grow: 1;
+  text-decoration: none;
+}
+
+.docs-title:hover {
+  color: #cccccc;
```

Comment:
that is the reason I dislike magenta: it only looks good with clean white and black, and it's terribly hard to find matching color (what I want to say - I dislike this gray)

Reply:
I did not find another visual hover effect which I like due to the fact that the blue looks ugly and a change of the background color is also a huge effect which consumes lot of space in the head sections. I thought it might be a good idea to have a light effect on the colorization of the font itself.

Reply:
nevertheless, we can skip the color. But I would like to have the link there.